src/openforms/forms/models/form.py

```diff
@@ -444,3 +446,37 @@ class FormLogic(models.Model):
         ),
         default=False,
     )
+
+
+class FormsExport(DeleteFileFieldFilesMixin, models.Model):
+    export_content = PrivateMediaFileField(
+        verbose_name=_("export content"),
+        upload_to="exports/%Y/%m/%d",
+        help_text=_("Zip file containing all the exported forms"),
+    )
+    downloaded = models.BooleanField(
+        verbose_name=_("downloaded"),
+        help_text=_("Whether the zip file has already been downloaded"),
+        default=False,
+    )
+    date_downloaded = models.DateField(
+        verbose_name=_("date downloaded"),
+        help_text=_("The date on which the zip file was downloaded"),
+        blank=True,
+        null=True,
+    )
+    user_email = models.EmailField(
+        verbose_name=_("user email"),
+        help_text=_(
+            "The email address that the user entered when requesting the export"
+        ),
+    )
+    username = models.CharField(
+        verbose_name=_("username"),
+        help_text=_("The unique username of the user that requested the download"),
+        max_length=150,
+    )
```

Comment:
Should be replaced with a single `user = models.ForeignKey(settings.AUTH_USER_MODEL)` and you can derive the username and email from those objects.

Reply:
I left the email field separate in case the user wants to enter a different email address or the email address associated with the user is blank.